python/src/xnat_mrd/fetch_datasets.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+import pooch
+from pathlib import Path
+
+
+def _set_up_zenodo_doi(base_url: str):
+    ZENODO = pooch.create(
+        path=Path(__file__).parents[3] / "test-data",
+        base_url=base_url,
+        registry=None,
+    )
```

Comment:
```suggestion
    ZENODO = pooch.create(
        path=Path(__file__).parents[3] / "test-data",
        base_url=base_url,
        registry=None,
        retry_if_failed=5
    )
```
You may want to enable [retrying failed downloads](https://www.fatiando.org/pooch/latest/sample-data.html#retry-failed-downloads) to avoid sporadic failures in CI?